scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -331,11 +348,17 @@ def default_autocast():
     for batch_idx, probs in enumerate(all_probs):
         preds = np.argmax(probs, axis=1)
         preds_tensor = torch.tensor(preds, device='cpu').unsqueeze(0)
-        pred_text = asr_model._wer.decoding.ctc_decoder_predictions_tensor(preds_tensor)[0][0]
+        if isinstance(asr_model, EncDecHybridRNNTCTCModel):
```

Comment:
If you save the right decoding into a variable in the top, then there is no need to check again here.